README.md

```diff
@@ -105,24 +128,32 @@ enzymes. This could have big implications for fields like genomics, synthetic bi
 and biocatalysis, which all rely on understanding how enzymes work.
 ```
 
+#### Using `SimpleBot` with a Local Ollama Model
+
 If you want to use an Ollama model hosted locally,
 then you would use the following syntax:
 
 ```python
 from llamabot import SimpleBot
+
+system_prompt = "You are Richard Feynman. You will be given a difficult concept, and your task is to explain it back."
 bot = SimpleBot(
-    "You are Richard Feynman. You will be given a difficult concept, and your task is to explain it back.",
+    system_prompt,
     model_name="ollama/llama2:13b"
 )
 ```
 
-Simply specify the `model_name` keyword argument
-and provide a model name from the [Ollama library of models](https://ollama.ai/library)
-prefixed by `ollama/`.
+Simply specify the `model_name` keyword argument following the `<provider>/<model name>` format. For example:
+
+* `ollama/` previx
```

Comment:
I think `previx` is mis-spelled, should be `prefix`, right?

Suggesting this change:

```suggestion
* `ollama/` as the prefix, and
```